### PR TITLE
adding kube-controller-manager starting option tests

### PR DESCRIPTION
**Pull request by huangjiuyuan**
**What this PR does / why we need it**:
The unit test for kube-controller-manager is missing in `cmd/kube-controller-manager/app/options`. I have added a unit test for checking kube-controller-manager starting options, which is similar to the unit test in `cmd/kube-apiserver/app/options/options_test.go`. This PR https://github.com/kubernetes/kubernetes/pull/49092 can be seen as a reference.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

`NONE`

### CODE REVIEW

**Comment by huangjiuyuan** [comment]
Hi, @wojtek-t 
Could you please take a look at this one?

**Comment by huangjiuyuan** [comment]
Hi @xiangpengzhao 
Could you please take a quick look at this?

File: cmd/kube-controller-manager/app/options/options_test.go [hunk 1]
```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```
Comment (xiangpengzhao) [inline cmd/kube-controller-manager/app/options/options_test.go]: 2017

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]
just one nit. otherwise LGTM.

**Comment by huangjiuyuan** [comment]
@xiangpengzhao Good to go?

**Comment by huangjiuyuan** [comment]
@wojtek-t

**Comment by huangjiuyuan** [comment]
@wojtek-t

**Comment by huangjiuyuan** [comment]
Hi, @sttts 
Could you please take a look at this? Thank you!

**Comment by huangjiuyuan** [comment]
@sttts Some tests failed and I had changed a little bit. Could you please approve again?

**Comment by huangjiuyuan** [comment]
@wojtek-t Could you please approve this one? It seems that This PR is still not approved.

**Pull request merged**